src/injected/styles/injected.scss

```diff
@@ -255,34 +255,70 @@
         }
     }
 
-    .insights-highlight-container .insights-highlight-box {
-        visibility: visible !important;
-        position: absolute !important;
-        overflow: hidden !important;
-        pointer-events: none !important;
+    .insights-highlight-container {
+        .insights-highlight-box {
+            visibility: visible !important;
+            position: absolute !important;
+            overflow: hidden !important;
+            pointer-events: none !important;
+        }
 
-        @include ms-high-contrast-override {
-            forced-color-adjust: none;
-            outline-color: Highlight !important;
+        // This applies to both the text in highlight boxes and also the text
+        // in highlight circled used for tab/focus visualizers
+        .insights-highlight-text {
+            @include ms-high-contrast-override {
+                // Note that forced-color-adjust: none is required for highlight box labels'
+                // outline/box-shadows to be inherited correctly in HC mode, in addition to being
+                // required for color and background-color
+                forced-color-adjust: none;
+                color: HighlightText !important;
+                background-color: Highlight !important;
+            }
         }
-    }
 
-    .insights-highlight-container .insights-highlight-text {
-        @include ms-high-contrast-override {
-            forced-color-adjust: none;
-            color: HighlightText !important;
-            background-color: Highlight !important;
+        .insights-highlight-box .insights-highlight-text {
+            cursor: default !important;
+            pointer-events: all !important;
+            font-size: 1em !important;
+            padding: 0.3em !important;
+            float: right !important;
+            position: relative !important;
+            text-align: center !important;
+
+            // This results in the highlight box outline appearing to also run down the left edge of
+            // the label
+            outline: inherit !important;
+            outline-offset: inherit !important;
+            box-shadow: inherit !important;
+        }
+
+        .insights-highlight-outline-solid {
+            // Non-HC outline-color is intentionally unset; individual formatters should override it
+            outline-width: 1px !important;
+            outline-style: solid !important;
+            outline-offset: 1px !important;
+            box-shadow: 0 0 0 3px white !important;
```

Comment:
Opinion: I find 1px outline over 3px shadow makes the red outline not stand out as much as I'm used to, especially around images. 
![image](https://user-images.githubusercontent.com/1752950/202820291-e5897020-190e-400c-8c22-449493ec9448.png)
I think 2px outline over 4px shadow is a little clearer:
![image](https://user-images.githubusercontent.com/1752950/202820206-540a6c20-e520-4311-8333-ce7fc501e2c9.png)
Will totally defer to @ferBonnin here.

Reply:
I see what you mean @peterdur specially for white backgrounds, I agree that 2px makes it more clearer but it also uses more space. I don't feel strongly about one way or another so if we prefer it, lets go with 2px with 4px shadow 🙂